docs/userguide/torch_forecasting_models.md

```diff
@@ -461,6 +461,109 @@ model.fit(...)
 
 *Note* : The callback will give one more element in the `loss_logger.val_loss` as the model trainer performs a validation sanity check before the training begins.
 
+#### Example with MLflow Autologging
+
+MLflow using interface (UI) and autologging to track Dart's pytorch models. 
+```python
+import pandas as pd
+from torchmetrics import MeanAbsolutePercentageError
```

Comment:
`import torchmetrics` as later torchmetrics is used directly (L486)

Reply:
Good idea.

Reply:
Added via commit c15eb1e9244d29cca103d81700f2d9ecf6c5ecf4
Thanks!